api/data_explorer/controllers/export_url_controller.py

```diff
@@ -103,6 +107,22 @@ def _get_entities_dict(cohort_name, query):
                 'query': query
             }
         })
+
+    for doc in doc_generator:
+        for sample in doc.get('samples', []):
+            sample_id = sample['sample_id']
+            export_sample = {
+                k.replace('.', '_'): v
```

Comment:
```
# TODO: Remove replace after new Entity Service is in prod.
```

Reply:
No longer relevant (done in indexer and we're not including dataset/table names).